src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -8,13 +8,23 @@
 
   <ItemGroup>
     <None Remove="Features\Schema\Migrations\1.sql" />
+    <None Remove="Features\Schema\Migrations\2.diff.sql" />
+    <None Remove="Features\Schema\Migrations\2.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\1.sql" />
+    <EmbeddedResource Include="Features\Schema\Migrations\2.diff.sql" />
+    <EmbeddedResource Include="Features\Schema\Migrations\2.sql" />
     <GenerateFilesInputs Include="Features\Schema\Migrations\1.sql" />
+    <GenerateFilesInputs Include="Features\Schema\Migrations\2.sql" />
     <Generated Include="Features\Schema\Model\V1.Generated.cs">
       <Generator>SqlModelGenerator</Generator>
       <Namespace>Microsoft.Health.Fhir.SqlServer.Features.Schema.Model</Namespace>
       <Args>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\1.sql'))</Args>
     </Generated>
+    <Generated Include="Features\Schema\Model\V2.Generated.cs">
+      <Generator>SqlModelGenerator</Generator>
+      <Namespace>Microsoft.Health.Fhir.SqlServer.Features.Schema.Model</Namespace>
+      <Args>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\2.sql'))</Args>
```

Comment:
I think once this is merged into master, we'll name this class VLatest instread of V2, then V2 when V3 becomes the latest

Reply:
Right, I agree that it makes sense to do this after merging to master. I have a work item for that [here](https://microsofthealth.visualstudio.com/Health/_workitems/edit/72041).